src/agent_IR_RaymondWang_LinYang.py

```diff
@@ -0,0 +1,98 @@
+#trying
+""" Information Ratio """
+import argparse
+import numpy as np
+from collections import deque
+from pedlar.agent import Agent
+import time as tm
+
+
+
+class IRAgent(Agent):
+    name = "IR"
+    def __init__(self, verbose=False, **kwargs):
+        self.verbose=verbose
+        self.slow = deque(maxlen=128)
+        self.fast = deque(maxlen=32)
+        self.last_order = None
+        self.price_list=[]
+        self.std=0
+        self.total_profit_list=[]
+        self.total_profit=0
+        self.order_in_progress=False
+        super().__init__(**kwargs)
+
+
+    # print order when placing an order
+    def on_order(self, order):
+        """On order handler."""
+        print('----------------------------------------------')
+        print("ORDER: ", order)
+        self.order_in_progress=True
+        self.last_order=order.id
+        self.price_list=[]
+
+    # print the profit after the order is closed
+    def on_order_close(self, order, profit):
+        """On order close handler."""
+        print("PROFIT:", profit)
```

Comment:
add:
if self.verbose:
    print(f"PROFIT: {profit}")

remove the print("-------------")